modules/win-autoscale/watchmaker-win-autoscale.template.cfn.yaml

```diff
@@ -475,11 +481,17 @@ Resources:
                 -
                   local_AssignInstanceRole: !If [AssignInstanceRole, !Sub '--role ${InstanceRole}', '']
                   local_UseCfnUrl: !If [UseCfnUrl, !Sub '--url ${CfnEndpointUrl}', '']
-            
               ignoreErrors: 'true'
               waitAfterCompletion: '0'
         install-cloudwatch-agent:
           commands:
+            01-manipulate-cloudwatch-config:
+              command: !Sub
+                - >-
+                  ${local_ShellCommandMap} -Command "
```

Comment:
Nit. Ending a line with a `"` in a `>-` multi-line yaml block means the command is going to be rendered as `" <command>..."` (note the space after first quote). Would suggest restructuring this as:

```
                  ${local_ShellCommandMap} -Command
                  "c:\cfn\scripts\AmazonCloudWatchAgent\cloudwatch-applog-config.ps1 -Verbose -ErrorAction Stop"
```